src/components/subscription-details/components/Edit/index.tsx

```diff
@@ -0,0 +1,6 @@
+import { h } from '@stencil/core';
+import PlanSelector from './PlanSelector';
+
+export const Edit = () => {
+  return <PlanSelector />;
```

Comment:
I think `PlanSelector` will eventually become stateless and use this function to grab the the relevant data from the store and pass it in here